provider/common/util.go

```diff
@@ -3,28 +3,24 @@
 
 package common
 
-import (
-	"fmt"
+import "fmt"
 
-	"github.com/juju/juju/environs"
-)
-
-// EnvFullName returns a string based on the provided environment
-// that is suitable for identifying the env on a provider. The resulting
-// string clearly associates the value with juju, whereas the
-// environment's UUID alone isn't very distinctive for humans. This
-// benefits users by helping them quickly identify in their hosting
-// management tools which instances are juju related.
-func EnvFullName(env environs.Environ) string {
-	modelUUID := env.Config().UUID()
+// ModelFullName returns a string based on the provided model
+// UUID that is suitable for identifying the env on a provider.
```

Comment:
Ehh, actually, I think it is an environ and not a model. It's the manifestation of a model. I should have left the function name alone.